plugins/quickbars.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: default
+title: Quick Toolbar plugin
+title_nav: Quick Toolbar
+description: User interface controls to create content faster.
+keywords: plugin inlite quickbar
+---
+
+The Quick Toolbar plugin (`quickbars`) enables new user interface components to help users create content faster. It can be used to create an experience similar to Medium, Quip, and other modern editing tools.
+
+It replaces the capabilities of the `inlite` theme in TinyMCE 4 or earlier.
+
+The Quick Toolbar plugin enables two new context toolbars:
+
+* _Quick Selection_ - shown when text is selected for quick access to formatting commands such as bold, italic and link. 
+* _Quick Insert_ - shown when a new line is created for the quick insertion of objects such as tables and images.
+
+It also enables three new toolbar controls:
+
+* _Quick Link_ - an inline experience for creating and editing links without a dialog
+* _Quick Image_ - immediately prompts a user to select a local image to upload
+* _Quick Table_ - immediately inserts a 2x2 table without prompting the user to select the number of rows and columns
+
+
+##### Example enabling both context toolbars with their default controls:
+
+```js
+tinymce.init({
+    selector: "div.tinymce",
+    plugins: [ 'quickbars' ],
+    toolbar: false,
+    menubar: false,
+    quickbars: true
```

Comment:
Should this have been changed from `inline: true` to `quickbars: true`? I suspect this may have accidentally been swapped since `inline` is very similar to `inlite`.